restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/release-notes.adoc

```diff
@@ -0,0 +1,47 @@
+= Restcomm Release Notes
+
+These notes group both Community and Telscale changes in a single document.
+Changes marked with (R) sign, means those changes are introduced for the respective
+Telscale commercial product.
+
+//add release-notes with newer on top
+== 8.4.0 version
+=== Additions 
+// New features (whether major or minor) go here
+* Clients passwords are now hashed in DB.
+* Profiles allow arbitrary properties to be saved/retrieved.
+* Added configurable SBC mode.
+
+=== Removals
+// draws attention to functionality that is getting removed
+* RVD auto-login from Console no longer works after hashed passwords
+
+=== Changes
```

Comment:
if these are linked to github issues, i think we should include the links

Reply:
There are no links to github issues